ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -244,6 +245,7 @@ a database ( directory )
           scope.database = true;
         }
         dbTracker.debugLog("database");
+        dbEventFound = true;
```

Comment:
should this be set to true only if scope.database = true. What is tracked using that?